cluster-autoscaler/README.md

```diff
@@ -27,10 +26,13 @@ there is a big chance that it won't work as expected.
 | 1.5.X  | 0.4.X  |
 | 1.4.X  | 0.3.X  |
 
-<sup>*</sup>Cluster Autoscaler 0.5.X is the official version shipped with k8s 1.6. We've done some basic tests using k8s 1.6 / CA 0.6 and we're not aware of any problems with this setup. However, CA internally simulates k8s scheduler and using different versions of scheduler code can lead to subtle issues.
+<sup>*</sup>Cluster Autoscaler 0.5.X is the official version shipped with k8s 1.6. We've done some basic tests using k8s 1.6 / CA 0.6 and we're not aware of any problems with this setup. However, Cluster Autoscaler internally simulates Kubernetes' scheduler and using different versions of scheduler code can lead to subtle issues.
 
 # Notable changes
 
+For CA 1.1.2 and later, please check [release
```

Comment:
Do we have releases on github for older versions? If so maybe we can completely drop this section. It's not too useful anyway and it's quickly getting outdated. I want to keep this short to avoid tl;dr effect of our FAQ. If we drop this maybe we could add some stuff that users often get confused about (I'd like some buildup on my "things that CA do / doesn't do" slides from deep dive on last kubecon, but I'm open to discussion :)). WDYT?

Reply:
There are some. I didn't read all of them, but they don't seem as extensive as the ones here. I suppose it's a question of how long we want to keep the history here.